plugins/kubernetes-backend/src/types/types.ts

```diff
@@ -27,7 +27,10 @@ export interface CustomResource {
 
 export interface ObjectFetchParams {
   serviceId: string;
-  clusterDetails: ClusterDetails;
+  clusterDetails:
+    | AWSClusterDetails
```

Comment:
Can we add `ClusterDetails` back here so that we can maintain backwards compatibility as least?

You'll need to re run the `api-reports` too after :)

Reply:
Thanks @benjdlambert . Changes are now live.